app/buyers/views/buyers.py

```diff
@@ -270,8 +278,225 @@ def view_brief_responses(framework_slug, lot_slug, brief_id):
+        csv_rows.append(column_headings)
+
+        # Add a row for each eligible response received
+        for brief_response in context['responses']:
+            if all_essentials_are_true(brief_response):
+                row = []
+                for key in question_key_sequence:
+                    if key in boolean_list_questions:
+                        row.extend(brief_response.get(key))
+                    else:
+                        row.append(brief_response.get(key))
+                csv_rows.append(row)
+
+        return Response(
+            csv_generator.iter_csv(csv_rows),
+            mimetype='text/csv',
+            headers={
+                "Content-Disposition": (
+                    "attachment;filename=responses-to-requirements-{}.csv"
+                ).format(brief['id']),
+                "Content-Type": "text/csv; header=present"
+            }
+        ), 200
+
+
+    def generate_ods(self, brief, responses):
+        doc = SpreadSheet()
+
+        doc.add_font(FontFace(name="Arial", fontfamily="Arial"))
+
+        doc.add_style("default", "table-cell", (
+            TableCellProperties(wrapoption="wrap", verticalalign="top"),
+            TextProperties(fontfamily="Arial", fontnameasian="Arial",
+                           fontnamecomplex="Arial", fontsize="11pt"),
+        ))
+
+        doc.add_style("header", "table-cell", (
+            TextProperties(fontfamily="Arial", fontnameasian="Arial",
+                           fontnamecomplex="Arial", fontsize="21pt", 
+                           fontweight="bold"),
+        ))
+
+        doc.add_style("title", "table-cell", (
+            TableCellProperties(wrapoption="wrap", verticalalign="top"),
+            TextProperties(fontfamily="Arial", fontnameasian="Arial",
+                           fontnamecomplex="Arial", fontsize="16pt", 
+                           fontweight="bold"),
+        ))
+
+        doc.add_style("subtitle", "table-cell", (
+            TextProperties(fontfamily="Arial", fontnameasian="Arial",
+                           fontnamecomplex="Arial", fontsize="16pt"),
+        ))
+
+        doc.add_style("intro", "table-column", (
+            TableColumnProperties(columnwidth="150pt", breakbefore="auto"),
+        ))
+
+        doc.add_style("response", "table-column", (
+            TableColumnProperties(columnwidth="300pt", breakbefore="auto"),
+        ))
+
+        doc.add_style("row", "table-row", (
+            TableRowProperties(useoptimalrowheight="true", breakbefore="auto"),
+        ))
+
+        sheet = doc.sheet("Supplier evidence")
+
+        content = content_loader.get_manifest(
+            brief['frameworkSlug'], 
+            'output_brief_response'
+        ).filter({'lot': brief['lotSlug']})
+
+        section = content.get_section('view-ods')
+
+        sheet.create_column(stylename="intro", defaultcellstylename="default")
+        sheet.create_column(stylename="intro", defaultcellstylename="default")
```

Comment:
It might be helpful to add a comment what data these columns contain (unless it's specified elsewhere that's really obvious).  I was just glancing through and I thought this might have been a copy+paste error.